server/eclipse-project/src/main/resources/sql/contest_comparison.sql

```diff
@@ -1,19 +1,19 @@
 -- For each contest, and for each cast vote record in the random sequence
--- (with duplicates) for which the Audit Board has submitted information, and which 
+-- (with duplicates) for which the Audit Board has submitted information, and which
 -- covers the contest in question,
 -- original cvr info, audit board interp info.
 -- note that the random sequence index (includes dupes) is contest_audit_info.index
 -- cvr_contest_info.index is the index of the *contest* on the ballot
--- Note that in case of an overvote, `cci_a.choices` shows all the choices the Audit Board thought the voter intended, while cci.choices will *not* show all those choices. 
+-- Note that in case of an overvote, `cci_a.choices` shows all the choices the Audit Board thought the voter intended, while cci.choices will *not* show all those choices.
 
```

Comment:
Is this TODO ours or CDOS IT's?

Reply:
That was already there!

Reply:
(And I think it probably is properly handled, because at least some PHANTOM_BALLOT items show up in this report.)